### ISSUE TITLE
Popover inside a portal doesn't get keyboard focus

### ISSUE THREAD

**Issue by Merott** [ycs77/headlessui-float#66]
**Use Version**
Use version when bugs appear:
 - Headless UI: v1.7.4
 - Headless UI Float: v0.11.2
 - Framework: react v16.8
 - `@floating-ui/core`: v1.0.0
 - `@floating-ui/dom`: v1.0.0
 - Browser: n/a

**Describe the bug**

When using headlessui-float with the `portal` option to render a popover, the popover doesn't receive keyboard focus as it should.

**To Reproduce**

[See this on StackBlitz](https://stackblitz.com/edit/github-dq8e3r?file=src%2FApp.tsx)

Toggle the use of a `portal` with the checkbox, and try navigating with your keyboard.

**Comment by ycs77** [comment]
Hi @Merott, I think this is not a bug, because using [Portal](https://react.dev/reference/react-dom/createPortal) will render the DOM element to another location in the page. Headless UI Float uses Headless UI's Portal component, which renders the element to the bottom of the `<body>`. The default tab-switching order is based on the DOM element order, so this phenomenon will appear after the Portal is enabled.

Here is a simple image to explain portaled DOM elements order:

![螢幕擷取畫面 2023-09-02 154618](https://github.com/ycs77/headlessui-float/assets/38133356/272a0bd2-d68a-4bdb-9493-aca00abc006b)

If you want to keep the tab order, it is recommended not to use Portal. If nested elements are hidden, you can check if the `overflow-hidden` class is added, deleting it will display it normally.

If you still need to use the Portal feature and keep the visual tab switching order, it is not within the scope of this package. It is recommended to [search for react portal tabindex](https://www.google.com/search?q=react+portal+tabindex) to find relevant solutions.

**Comment by Merott** [comment]
@ycs77 Thanks for the detailed response. I agree it seems outside the scope of this package.
